package.json

```diff
@@ -6,10 +6,14 @@
     "start": "ng serve",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "ng test --browsers=ChromeHeadless --watch=false --code-coverage",
+    "lint": "node ./node_modules/eslint/bin/eslint src --ext .js,.jsx,.ts,.tsx",
+    "precommit": "pretty-quick --staged",
+    "prepare": "husky install .husky"
   },
   "private": true,
   "dependencies": {
+    "@angular-eslint/schematics": "^15.2.1",
```

Comment:
Hmm, is this not also just a dev dependency?

Reply:
True, done